cmd/csi-attacher/main.go

```diff
@@ -179,18 +187,26 @@ func main() {
 	if !*enableLeaderElection {
 		run(context.TODO())
 	} else {
-		// Leader election was requested.
-		if leaderElectionNamespace == nil || *leaderElectionNamespace == "" {
-			klog.Error("-leader-election-namespace must not be empty")
+		var le leaderElection
+
+		// Name of config map with leader election lock
+		lockName := "external-attacher-leader-" + csiAttacher
+		if *leaderElectionType == "configmaps" {
+			le = leaderelection.NewLeaderElectionWithConfigMaps(clientset, lockName, run)
```

Comment:
Can you also print out a warning message saying that the configmap option is deprecated.

Reply:
done